docs/api/tools/block-explorer/block-view.md

```diff
@@ -1,58 +1,50 @@
 # Browse blocks
 
-![Browse blocks!](../../../assets/images/block-tx.png "View blocks on zkSync")
+![Browse blocks!](../../../assets/images/blocks.png "View blocks on zkSync")
 
-The block explorer home screen, by default, displays the 10 most recent blocks in the chain.
-View all blocks for a more complete overview of recent blocks, or click on a specific block number for details on that block.
-The blocks page is also accessible via the top menu.
+The block explorer home screen shows the 10 most recent blocks by default. To see a more complete overview of recent blocks, you can view all blocks. You can also click on a specific block number to see details about that particular block.
+You can access the blocks page from the top menu as well.
```

Comment:
```suggestion
- The block explorer home screen shows the 10 most recent blocks by default. 
To see a more complete overview of recent blocks, you can view all blocks. You can also click on a specific block number to see details about that particular block.
You can access the blocks page from the top menu as well.
```

Reply:
Link to block explorer?

This sentence **- You can also access the blocks page from the top menu.** should be in the previous section as an introduction to the pic and should be something like:

**Access the blocks page from the top menu in the explorer dashboard**

Then you can go into the specifics.